.github/workflows/odroid_debug_arm64.yml

```diff
@@ -29,9 +29,8 @@ jobs:
       - name: test
         env:
           CMAKE_BUILD_TYPE: Debug
-          # FIXME: switched off backtraces support till #6142 is fixed,
-          #        disabled '-Wtype-limits' flag till #6143 is fixed.
-          CMAKE_EXTRA_PARAMS: '-DENABLE_BACKTRACE=OFF -DCMAKE_C_FLAGS="-Wno-type-limits "'
+          # FIXME: switched off backtraces support till #6142 is fixed
```

Comment:
Nit: missed dot at the end of the sentence.

Reply:
Fixed